### ISSUE TITLE
Carthage support

### ISSUE THREAD

**Issue by ghost** [ninjaprox/NVActivityIndicatorView#13]
Opening a new issue as requested by @ninjaprox:

Dependency "NVActivityIndicatorView" has no shared framework schemes

If you believe this to be an error, please file an issue with the maintainers at https://github.com/ninjaprox/NVActivityIndicatorView/issues/new

**Comment by ninjaprox** [comment]
@morizotter: Could you help checking this?

**Comment by morizotter** [comment]
Will be check later in today!

**Comment by morizotter** [comment]
Mmm..

In my project currently I'm using this with statement below and it succeeds with no error. I tried to do it now and the result is same, succeeds.

```
github "morizotter/NVActivityIndicatorView" "morizotter/carthage-compatible"
```

And then, I tried to `carthage update` with `github "ninjaprox/NVActivityIndicatorView" == 1.3`

It fails... mmm.

**Ah, 1.3 was not compatible with Carthage**.

**Comment by morizotter** [comment]
Ah, I make this pull request when this project was Swift1.2. And now the project is Swift2.0 and Xcode7.0. This pull request merged after update. I think it is the problem...

**Comment by morizotter** [comment]
I think @iagomr maybe using NVActivityIndicatorView v1.3. I got same error.

NVActivityIndicatorView is compatible with Carthage from v2.0. So you have to use it with Xcode7.0 or later.

In my environment, I have Xcode7.1(Swift2.1). So, I tried to make [NVActivityIndicatorView for Swift2.1](https://github.com/morizotter/NVActivityIndicatorView/tree/morizotter/fix-carthage) and use it from Xcode7.1 sample project. 

```
github "morizotter/NVActivityIndicatorView" "morizotter/fix-carthage"
```

It works.

**Comment by ghost** [comment]
Hey @morizotter,

I tried it with your latest version last week, but I noticed I was using an older version of Carthage (something like 0.8). I just used your framework manually so "it's working" now.

I upgraded to Carthage 0.9.3 and will try your framework again this week, perhaps there's no problem anymore. 

Thanks for looking into it!

**Comment by ninjaprox** [comment]
It's maybe because I haven't release new version yet. I have updated latest version and test myself, everything works well.
@iagomr: Could you try with `github "ninjaprox/NVActivityIndicatorView"` in `Cartfile`? Let me know if the problem is till there.

**Comment by morizotter** [comment]
Ah! You are right ><  @ninjaprox I am mistaken that latest version is v2.0 ><.

**Comment by ninjaprox** [comment]
No, it's not you mistake. I've just released version `v2.1`. Eventually, solved this issue, will close this as well. Thanks for your report @iagomr.
